```
 error[E0308]: mismatched types
   --> /home/runner/work/servo/servo/target/aarch64-unknown-linux-ohos/checked-release/build/servo-script-bindings-7f9a62dce65d4725/out/Bindings/XRWebGLSubImageBinding.rs:267:15
    |
267 |         name: c"XRWebGLSubImage".as_ptr(),
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*const i8`, found `*const u8`
    |
    = note: expected raw pointer `*const i8`
               found raw pointer `*const u8`
```

--- a/components/script_bindings/init.rs
+++ b/components/script_bindings/init.rs
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+use std::ffi::c_char;
 use std::ptr;
 
 use js::jsapi::{
@@ -37,7 +38,7 @@ pub(crate) struct InitClassOpsConfig {
 }
 
 pub(crate) struct DomJSClassConfig {
-    pub(crate) name: *const i8,
+    pub(crate) name: *const c_char,
     pub(crate) flags: u32,
     pub(crate) slots: u32,
     pub(crate) class: DOMClass,

